app/design/frontend/Magento/luma/Magento_Customer/layout/customer_account.xml

```diff
@@ -7,12 +7,6 @@
 -->
 <page layout="2columns-left" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
     <body>
-        <referenceBlock name="sidebar.main.account_nav">
-            <arguments>
-                <argument name="block_title" translate="true" xsi:type="string">My Account</argument>
-                <argument name="block_css" xsi:type="string">block-collapsible-nav</argument>
-            </arguments>
-        </referenceBlock>
         <move element="page.main.title" destination="content.top" before="-"/>
```

Comment:
Can't this whole file be deleted / moved-into-blank-theme? You also put the `<move />` in Blank, so no need to leave it here since the xml will be merged.